### PR TITLE
Update EBNF

### PR DESCRIPTION
**Pull request by cmontella**

### CODE REVIEW

File: src/handbook/ebnf.md [hunk 1]
```diff
@@ -30,22 +30,23 @@ numeric = "0" .. "9";
 number = ["-"] {numeric} ["." {numeric}];
 string-interpolation = "{{" expression "}}";
 string = "\"" {string-interpolation | unicode - "\"" | "\\\"" | whitespace} "\"";
-uuid ="⦑" (unicode - specials)  "⦒"
+uuid ="⦑" (unicode - specials)  "⦒";
```
Comment (convolvatron) [inline src/handbook/ebnf.md]: i think we agreed that this syntax is gone, but need to check w/ ibdknox

File: src/handbook/ebnf.md [hunk 2]
```diff
@@ -69,11 +70,11 @@ function = identifier "[" [attribute] {whitespace+ attribute} "]";
 
 ```ebnf
 record = "[" [attribute] {whitespace+ attribute} "]"
-attribute = name | tag | attribute-not | identifier {whitespace+ comparator whitespace+ expression};
+attribute = tag | attribute-not | identifier {whitespace+ comparator whitespace+ expression};
 name = "@" (identifier | string);
```
Comment (convolvatron) [inline src/handbook/ebnf.md]: should probably be dbname or some such

File: src/handbook/ebnf.md [hunk 3]
```diff
@@ -92,51 +93,51 @@ expression = number | identifier | function | infix | record | attribute-access;
 ## Statements
 
 ```ebnf
-comment = "//" {unicode | whitespace - newline} newline
-statement = record | function | not-statement | if-statement | comparison | comment
+comment = "//" {unicode | whitespace - newline} newline;
+statement = record | function | not-statement | if-statement | comparison | comment;
```
Comment (convolvatron) [inline src/handbook/ebnf.md:96]: is?
where is comparison defined?

File: src/handbook/ebnf.md [hunk 4]
```diff
@@ -92,51 +93,51 @@ expression = number | identifier | function | infix | record | attribute-access;
...
+attribute-update = attribute-access whitespace+ (":=" | "+=" | "-=") whitespace+ expression;
+update-operation = create | delete | merge | tag-update | attribute-update;
 ```
 
 ## If-Then
 
 ```ebnf
-group = "(" expression {whitespace+ expression} ")"
-binding-group = "(" identifier {whitespace+ identifier} ")"
+group = "(" expression {whitespace+ expression} ")";
```
Comment (convolvatron) [inline src/handbook/ebnf.md:113]: maybe take care of if in this pass?

**Review by convolvatron — COMMENTED** [review]
github, i don't understand what this process is supposed to look like. my feedback is that you made a really weird review tool

see comments

File: src/handbook/ebnf.md (hunk 1 above)
Comment (cmontella) [reply src/handbook/ebnf.md]: @ibdknox ^

Comment (ibdknox) [reply src/handbook/ebnf.md]: yeah, I think we can just get rid of it.

File: src/handbook/ebnf.md (hunk 3 above)
Comment (cmontella) [reply src/handbook/ebnf.md:96]:
```
is-expression = is "(" comparison ")";
comparison = expression whitespace+ comparator whitespace+ expression;
```

File: src/handbook/ebnf.md (hunk 4 above)
Comment (cmontella) [reply src/handbook/ebnf.md:113]: What's wrong with if?

File: src/handbook/ebnf.md (hunk 3 above)
Comment (convolvatron) [reply src/handbook/ebnf.md:96]: sorry, they used the damn foldy thing on me

**Review by convolvatron — APPROVED** [review]
the remaining two issues were bogus

File: src/handbook/ebnf.md [hunk 5]
```diff
@@ -92,51 +92,51 @@ expression = number | identifier | function | infix | record | attribute-access;
...
-else-expression = else whitespace+ if-result
+else-expression = else whitespace+ if-result;
 if-statement = (identifier | binding-group) whitespace+ equality whitespace+
                if-expression
                {whitespace+ (if-expression | else-if-expression)}
-               [else-expression]
+               [else-expression];
 ```
 
 ## Sections
```
Comment (convolvatron) [inline src/handbook/ebnf.md:125]: if you see this section should go to action

**Review by convolvatron — DISMISSED** [review]

**Pull request closed without merging**